### ISSUE TITLE
[Support Admin] Add organization admin should have an H1

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6400]
## Area or Component

Support Admin > Add organization Admin

## Quick Summary

There is no H1 on the page
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/ad0d9281-bddc-4f08-a277-1d8384b2a2ee)

## Accessibility Violation

https://dequeuniversity.com/rules/axe/4.7/page-has-heading-one?application=AxeChrome

## Action Requested

Change the header on the page,`Add organization admin`, to be h1.

## Source Code (optional)

[which file or line is the issue attached to?]: #
